### ISSUE TITLE
Wizard

### ISSUE THREAD

**Issue by CJ-Wright** [regro/rever#70]
We may want a wizard (ala xonsh setup) for setting up the `rever.xsh` file.

**Comment by scopatz** [comment]
Yeah, this would be nice.  Not sure I am up for it. Xonsh has a whole toolkit for creating such wizards though.

**Comment by scopatz** [comment]
So it wouldn't be so hard to do.
